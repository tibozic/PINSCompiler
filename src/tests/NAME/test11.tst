var x : integer;
var y : x
