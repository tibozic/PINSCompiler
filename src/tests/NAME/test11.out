[2:1-2:10]: ERROR: Variable type `x` is not a type

